gh_pages/_source/tesseract_motion_planning_api_doc.rst

```diff
@@ -0,0 +1,168 @@
+*****************************
+Tesseract Motion Planning API
+*****************************
+
+Overview
```

Comment:
I think I might consider restructuring this. Rather than just stepping through a tesseract_ros example, it might be more useful to describe the pieces of a motion planning system and how we will interact with them.

I think I'd give an overview of the pieces like this with links to more details

- Environment Monitor - Description and link to page for modifying the environment
- Planning Server - Description
- Taskflows - What are available out of the box, and how do you define your own? Probably its own page too
- Command language - How to define a simple program. Link to more detailed docs

Then a simple example (e.g. <100 lines) that shows how to load a urdf, setup the planning server, and send it a request.The freespace_manager_example in tesseract_process_managers would probably be a good one to go off of.

Reply:
I agree, I think that would be more helpful to have each component described and to use a simpler example that doesn't have other functionality distracting from the motion planning concept.

I rewrote this page in 81e5c20. It still needs some work, but it's an improvement from the last version. I used the example you pointed out as well. For now, I've omitted the description of taskflow until I have a better understanding of how it works and can describe it sufficiently.

Let me know what you think of this version.